microsoft-edge/web-platform/release-notes/136.md

```diff
@@ -0,0 +1,498 @@
+
+
+<!-- ---------- -->
+###### Deprecate `Intl.Locale` getters
+
+The `Intl.Locale` API exposes information for a locale, including its week elements and its hour cycle type, such as 12-hour clock, Japanese 12-hour clock, or 24-hour clock.
+
+The accessor properties of the `Intl.Locale` object are now deprecated, in favor of the corresponding functions, per the specification.  For example, the `Intl.Locale.prototype.hourCycle` accessor property is deprecated in favor of the `Intl.Locale.prototype.getHourCycles()` function.
+
+See [Intl.Locale](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale) at MDN.
+
+
+<!-- ====================================================================== -->
+## Origin trials
+
+The following are new experimental APIs that you can try on your own live website for a limited time.
+
+To learn more about origin trials, see [Use origin trials in Microsoft Edge](../../origin-trials/index.md).
+
+To see the full list of available origin trials, see [Microsoft Edge Origin Trials](https://developer.microsoft.com/microsoft-edge/origin-trials/).
+
+
+<!-- ------------------------------ -->
+#### New Chromium origin trials
+
+
+<!-- ---------- -->
+###### Update of Canvas text rendering implementation
+
+Expires on December 16, 2025.
+
+New implementation of `measureText()`, `fillText()`, and `strokeText()` for the CanvasRenderingContext2D API.  You can compare the performance of these methods against the previous implementation of these methods.
+
+* [Explainer](https://docs.google.com/document/d/1UbHuoflNyLbSavpH86HEmXFGyz2V4uZ6cjNpuuwrtBg/edit?usp=sharing)
+* [Feedback](https://crbug.com/new)
+* [Register](https://developer.chrome.com/origintrials/#/register_trial/3227392082964381697)
+
+
+<!-- ---------- -->
+###### Set preferred audio output device
+
+Expires on January 27, 2026.
+
+This feature allows a page to set a non-default device for all audio output from the page, including its child frames.
+
+* [Explainer](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/SetPreferredSinkId/explainer.md)
+* [Feedback](https://github.com/MicrosoftEdge/MSEdgeExplainers/issues)
+* [Register](https://developer.chrome.com/origintrials/#/register_trial/1173750652883435521)
+
+
+<!-- ---------- -->
+###### `confidence` field on `PerformanceNavigationTiming` object
+
+Expires on October 14, 2025.
+
+The `PerformanceNavigationTiming` object now includes a `confidence` field.  Use the `confidence` field to evaluate whether navigation timings accurately reflect your web app's performance.  The data initially contains some noise, because this field employs differential privacy, to safeguard user privacy.  By gathering this field in your Real User Monitoring (RUM) data, you can accumulate enough records to analyze trends and filter out the noise, gaining meaningful insights over time.
```

Comment:
The wording change, the last sentence doesn't represent what a developer would do. So perhaps this?


By gathering this field in your Real User Monitoring (RUM) data, you can filter out the noise with enough records to analyze trends and gain meaningful insights over time.

Reply:
might change last 'with' to 'by using'
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/u/using-by-using-with